R/dplyr.R

```diff
@@ -51,11 +51,7 @@ bind_rows <- function(x, y = NULL) {
       y[[nme]] <- NA
     }
   }
-  rbind.data.frame(x, y)
-}
-
-if_else <- function(condition, true, false, missing = NULL) {
-  ifelse(condition, true, false)
+  rbind.data.frame(x, y, ...)
```

Comment:
What's the point in changing the API here? Do you support more than two arguments? In this case it might be slightly better to forward to `bind_rows(rbind.data.frame(x, y), ...)` .

Reply:
Yes that was the idea.